### PR TITLE
Fix acknowledgement targets dependencies issues that would produce in…

### PR DESCRIPTION
**Pull request by rahul-malik**
…valid bazel

target labels

### CODE REVIEW

File: Sources/PodToBUILD/ObjcLibrary.swift [hunk 1]
```diff
@@ -682,6 +684,12 @@ private func extractSources(patterns: [String]) -> [String] {
     }
 }
 
+private func extractSwiftSources(patterns: [String]) -> [String] {
```
Comment (jerrymarino) [inline Sources/PodToBUILD/ObjcLibrary.swift]: Is this change intentional?

File: Sources/PodToBUILD/BuildFile.swift [hunk 2]
```diff
@@ -140,10 +140,12 @@ struct InsertAcknowledgementsTransform: SkylarkConvertibleTransform {
                     return [target]
                 }
 
+
                 let deps = target.acknowledgedDeps ?? [String]()
+                let externalDeps = deps.filter { $0.hasPrefix("//") }
                 let acknowledgement = AcknowledgmentNode(name: target.name,
                                                          license: podSpec.license,
-                                                         deps: deps)
```
Comment (jerrymarino) [inline Sources/PodToBUILD/BuildFile.swift]: If depend on the subspec, `//IGListKit:Core`, will this correctly propagate all licensing? I thought for some reason, this was required.

File: IntegrationTests/GoldMaster/Texture.podspec.json.goldmaster [hunk 3]
```diff
@@ -234,9 +234,7 @@ objc_library(
 acknowledged_target(
   name = "PINRemoteImage_acknowledgement",
   deps = [
-    "//Vendor/PINRemoteImage:PINCache_acknowledgement",
-    ":Core_acknowledgement",
```
Comment (jerrymarino) [inline IntegrationTests/GoldMaster/Texture.podspec.json.goldmaster]: In the case that a user depended on a top level target, say for example `PINRemoteImage`, but didn't explicitly depend on `PINCache`, would the license of `PINCache` still be propagated with these lines removed? I thought this is how licenses were trans propagated.

File: Sources/PodToBUILD/BuildFile.swift (hunk 2 above)
Comment (rahul-malik) [reply Sources/PodToBUILD/BuildFile.swift]: Yes since the other pod will begin with `//Vendor/IGListKit`

File: IntegrationTests/GoldMaster/Texture.podspec.json.goldmaster (hunk 3 above)
Comment (jerrymarino) [reply IntegrationTests/GoldMaster/Texture.podspec.json.goldmaster]: @rahul-malik - did you address this issue in the latest revision? From my understanding, the trans licenses will not propagate anymore.
